src/tests/system/tests/test_authentication.py

```diff
@@ -13,7 +13,12 @@
 
 @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
 @pytest.mark.parametrize("method", ["su", "ssh"])
-def test_authentication__login(client: Client, provider: GenericProvider, method: str):
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
+@pytest.mark.require(
+    lambda client, sssd_service_user: ((sssd_service_user == "root") or client.features["non-privileged"]),
+    "SSSD was built without support of running under non-root"
```

Comment:
Shouldn't it be "support **for** running..."?

Reply:
Maybe. Prepositions are difficult.
@justin-stephenson, could you please help here?

Reply:
`support for running` is better, alternatively the below options are clear IMO.

`SSSD built without non-root support`

`SSSD was built without non-privileged support`

Reply:
Updated.